### ISSUE TITLE
original fire data to package fillcell

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#816]
Import the original fire data to package fillcell and implement an example that creates such CellularLayer from scratch.

**Comment by pedro-andrade-inpe** [comment]
#813.

**Comment by pedro-andrade-inpe** [comment]
The error was:
```lua
SWIG_SystemError:boost::filesystem::create_directory: O sistema nÒo pode encontr
ar o caminho especificado
Stack traceback:
    File C:/TerraME/bin/packages/terralib/lua/TerraLib.lua, line 601, in functio
n 'createCellSpaceLayer'
    File C:/TerraME/bin/packages/terralib/lua/TerraLib.lua, line 961, in functio
n 'addShpCellSpaceLayer'
    File C:/TerraME/bin/packages/terralib/lua/Project.lua, line 232, in function
 'addCellularLayer'
    File C:/TerraME/bin/packages/fillcell/examples/emas.lua, line 60, in main ch
unk
```

**Comment by pedro-andrade-inpe** [comment]
The error above was solved by #1043.
